django/contrib/admin/templates/admin/base_site.html

```diff
@@ -1,6 +1,6 @@
 {% extends "admin/base.html" %}
 
-{% block title %}{% if subtitle %}{{ subtitle }} | {% endif %}{{ title }} | {{ site_title|default:_('Django site admin') }}{% endblock %}
+{% block title %}{% if errors %}{{_('Error:')}} {% endif %}{% if subtitle %}{{ subtitle }} | {% endif %}{{ title }} | {{ site_title|default:_('Django site admin') }}{% endblock %}
```

Comment:
```suggestion
{% block title %}{% if errors %}{{ _('Error:') }} {% endif %}{% if subtitle %}{{ subtitle }} | {% endif %}{{ title }} | {{ site_title|default:_('Django site admin') }}{% endblock %}
```
(I think this is a styling convention)